### PR TITLE
docs: refactor environment variables document after encapsulation of environment varibles

### LINKED ISSUE #142: Abstraction and standardisation of environment variables
EPIC https://github.com/tazama-lf/frms-coe-lib/issues/186

https://github.com/tazama-lf/docs/blob/dev/Research-Articles/Encapsulation-of-environment-variables.md

## Acceptance criteria

 - Updated documentation - env-var register

### PR DESCRIPTION
**Pull request by cshezi**
# SPDX-License-Identifier: Apache-2.0

## What did we change?
Refactored docs page to reflect the new changes of encapsulating environment variables

## Why are we doing this?
To allow integrators to have easier time using the library

## How was it tested?
- [ ] Locally
- [ ] Development Environment
- [ ] Not needed, changes very basic
- [ ] Husky successfully run
- [ ] Unit tests passing and Documentation done

### CODE REVIEW

File: Research-Articles/Environment-Variables-Starndazation.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+| TMS | event-director | rule-executer | typology-processor | TADP |
+| -------- | ------- | ------- | ------- | ------- |
+| LOGSTASH_HOST | LOGSTASH_HOST | LOGSTASH_HOST | LOGSTASH_HOST | LOGSTASH_HOST |
+| LOGSTASH_PORT | LOGSTASH_PORT | LOGSTASH_PORT | LOGSTASH_PORT | LOGSTASH_PORT |
+| LOGSTASH_LEVEL | LOGSTASH_LEVEL | LOGSTASH_LEVEL | LOGSTASH_LEVEL | LOGSTASH_LEVEL |
+| SIDECAR_HOST | SIDECAR_HOST | | SIDECAR_HOST | SIDECAR_HOST |
+
+# Recomendations - Short Term
+
+- Unify env naming conventions accross all processors - maybe use the library to store config classes and validate provided config.
```
Comment (rtkay123) [inline Research-Articles/Environment-Variables-Starndazation.md]:
```suggestion
- Unify env naming conventions across all processors - maybe use the library to store config classes and validate provided config.
```

File: Research-Articles/Environment-Variables-Starndazation.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+
+  Pros
+    - Will be able to use the same environmental variables throughout the solution.
+    - Single verify function to be called, can use optional parameters to determine what to check
+
+  Cons
+    - Will requite some way to determine which processors require which config.
```
Comment (rtkay123) [inline Research-Articles/Environment-Variables-Starndazation.md]:
```suggestion
    - Will require some way to determine which processors require which config.
```

File: Research-Articles/Environment-Variables-Starndazation.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+
+- When validating environmental variables, use diffrent levels , ie - Error message when database is missing, but warning if nats env is missing.
```
Comment (rtkay123) [inline Research-Articles/Environment-Variables-Starndazation.md]:
```suggestion
- When validating environmental variables, use different levels, i.e - Error message when database is missing, but warning if NATS env is missing.
```

**Review by rtkay123 — COMMENTED** [review]
I left some suggestions

**Review by rtkay123 — DISMISSED** [review]

**Review by Sandy-at-Tazama — CHANGES_REQUESTED** [review]
Please fix the name of the file - `standardization`

**Review by Sandy-at-Tazama — DISMISSED** [review]

**Review by Sandy-at-Tazama — APPROVED** [review]

**Pull request merged**